src/pages/ProfilePage.js

```diff
@@ -108,7 +108,7 @@ function ProfilePage(props) {
 
     // If we have a reportID param this means that we
     // arrived here via the ParticipantsPage and should be allowed to navigate back to it
-    const shouldShowLocalTime = !ReportUtils.hasAutomatedExpensifyEmails([login]) && !_.isEmpty(timezone);
+    const shouldShowLocalTime = !ReportUtils.hasAutomatedExpensifyEmails([accountID]) && !_.isEmpty(timezone);
```

Comment:
this is still using emails

Reply:
I'll instead use `hasAutomatedExpensifyAccountIDs here`